Jobs/src/Job.IdentityDeletion/Program.cs

```diff
@@ -97,6 +98,11 @@ static IHostBuilder CreateHostBuilder(string[] args)
                 .AddModule<SynchronizationModule>(configuration)
                 .AddModule<TokensModule>(configuration);
 
+            services.AddMediatR(c => c
+                .RegisterServicesFromAssemblyContaining<CancelStaleIdentityDeletionProcessesCommand>());
```

Comment:
Can be removed. This is called in `AddModule<DevicesModule>`

Reply:
Changes made.